udata/core/organization/commands.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+from udata.commands import manager
+from udata.models import GeoZone, Organization
+
+log = logging.getLogger(__name__)
+
+
+@manager.command
+def attach_zone(insee_code, organization_id_or_slug, level='fr/town'):
```

Comment:
It introduces a bit of complexity, let's do that when we'll introduce more levels.